nixos/modules/services/networking/smokeping.nix

```diff
@@ -339,14 +339,10 @@ in
       };
       preStart = ''
         mkdir -m 0755 -p ${smokepingHome}/cache ${smokepingHome}/data
-        rm -f ${smokepingHome}/cropper
-        ln -s ${cfg.package}/htdocs/cropper ${smokepingHome}/cropper
-        rm -f ${smokepingHome}/css
-        ln -s ${cfg.package}/htdocs/css ${smokepingHome}/css
-        rm -f ${smokepingHome}/js
-        ln -s ${cfg.package}/htdocs/js ${smokepingHome}/js
-        rm -f ${smokepingHome}/smokeping.fcgi
-        ln -s ${cgiHome} ${smokepingHome}/smokeping.fcgi
+        ln -sf ${cfg.package}/htdocs/cropper ${smokepingHome}/cropper
```

Comment:
It doesn't look like `${cfg.package}/htdocs/cropper` exists at all. This can probably be removed entirely?

Reply:
:rabbit2:

Reply:
Yeah, that path does not exist. Dropped it.